presentation/src/main/java/me/androidbox/presentation/navigation/NavigationGraph.kt

```diff
@@ -65,5 +68,19 @@ fun NavigationGraph(
                 },
             )
         }
+
+        /* Agenda Screen */
+        composable(
+            route = Screen.AgendaScreen.route
+        ) {
+            val agendaViewModel: AgendaViewModel = hiltViewModel()
+            val agendaScreenState = agendaViewModel.agendaScreenState.collectAsState()
```

Comment:
Recently the collectAsStateWithLifecycle() dependency got stable, so I'd recommend to use that since it's the safest way to collect flows in Android (currently, your flow will be collected even if the app is in the background). For that, you'll need this dependency:
```
implementation "androidx.lifecycle:lifecycle-runtime-compose:2.6.0"
```
Then use it like this:
```
val state by viewModel.state.collectAsStateWithLifecycle()
```

Reply:
@philipplackner Updated to use collectAsStateWithLifecycle for all collections in the navigationGraph